src/controller/LottoCtrl.js

```diff
@@ -0,0 +1,105 @@
+const {
+  BUDGET_MESSAGE,
+  TICKET_MESSAGE,
+  WINNING_NUMBER_MESSAGE,
+  TICKET_NUMBER,
+  BONUS_MESSAGE,
+  LOTTO_RANK,
+  STATIC_MESSAGE,
+} = require('../constants/lotto');
+const GameCtrl = require('./GameCtrl');
+const LottoView = require('../view/LottoView');
+const LottoModel = require('../model/LottoModel');
+
+const LottoCtrl = class extends GameCtrl {
+  constructor() {
+    super(new LottoView(), new LottoModel());
+  }
+
+  gameProcess() {
+    this.inputLottoBudget();
+  }
+
+  inputLottoBudget() {
+    const onInputLottoBudget = budget => {
+      this.model.setLottoBudget(budget);
+      this.buyLottoTicket();
+    };
+
+    this.view.input(BUDGET_MESSAGE, onInputLottoBudget);
+  }
+
+  buyLottoTicket() {
+    const ticketCount = this.getLottoCount();
+
+    const lottoTickets = [...new Array(ticketCount)].reduce(tickets => {
+      const currTicket = this.model.pickLottoTickets(TICKET_NUMBER);
+      tickets.push(currTicket);
+
+      return tickets;
+    }, []);
+
+    this.model.setLottoTickets(lottoTickets);
+    this.renderLottoTickets(lottoTickets);
+
+    this.inputLottoWinningNumbers();
+  }
+
+  renderLottoTickets(lottoTickets) {
+    lottoTickets.forEach(ticket => {
+      this.view.output(`[${ticket.join(', ')}]`);
+    });
+  }
+
+  getLottoCount() {
+    const ticketCount = this.model.setLottoCount();
+    const ticketMessage = `${ticketCount}${TICKET_MESSAGE}`;
+
+    this.view.output(ticketMessage);
+    return ticketCount;
+  }
+
+  inputLottoWinningNumbers() {
+    const onGetLottoWinningNumbers = winningNumbers => {
+      winningNumbers = winningNumbers.split(',').map(number => Number(number));

```

Comment:
조금 더 간략히 표현할 수 있을 것 같아요!

```
winningNumbers = winningNumbers.split(',').map(number => Number);
```

Reply:
```suggestion
      winningNumbers = winningNumbers.split(',').map(Number);
```
혹시 이 코드 말씀하신 것 아닐까요?

Reply:
헉 잘못 표기했네요.. 이런 실수를 하다니.. 죄송합니다 😥 
@solo5star 정정해주셔서 감사합니다!

Reply:
@solo5star @Leemainsw 감사합니다!! ㅎㅎ 해당 방식으로 수정했습니다!! 😁😁😁